rlschool/A1_robot/robots/a1_robot_velocity_estimator.py

```diff
@@ -0,0 +1,106 @@
+"""Estimates base velocity for A1 robot from accelerometer readings."""
```

Comment:
I do not see any file using this estimator. 
Is it necessary for the simulator?